README.txt

```diff
@@ -1,27 +1,30 @@
-#Initially created a clean installation of the latest python 3.11
+#created a clean installation of python 3.11
 
 #installed/upgraded pip using the command:
-py -m pip install --upgrade pip
+    py -m pip install --upgrade pip
 
 #Created a new virtual environment using the command:
-py -m venv env
+    py -m venv env
+
+#Activated the virtual environment using the command:
+    .\env\Scripts\activate
 
 #verified I was using the virtual environment using the command:
-where python
+    where python
 
 # Installed Jupyter Notebook using the command:
-pip install notebook
+    pip install notebook
 
 #Created a requirements.txt file using the command:
- pip freeze > requirements.txt
+    pip freeze > requirements.txt
+this generates a list of dependencies in the terminal that are copied into a text file called
+ "requirements.txt"
```

Comment:
It should be requirements intallation, not freezing in readme. I cannot install the project by following this instructions.